src/test/addrman_tests.cpp

```diff
@@ -398,8 +405,8 @@ BOOST_AUTO_TEST_CASE(addrman_getaddr)
     // Test 25: Ensure GetAddr still returns 23% when addrman has many addrs.
     for (unsigned int i = 1; i < (8 * 256); i++) {
         int octet1 = i % 256;
```

Comment:
Can this be achieved by fixing octet2 to zero and then setting octet3 to `i >> 8 % 256`?